### PR TITLE
[Messenger] Fix middleware docblocks

### PR DESCRIPTION
**Pull request by pamil**
| Q             | A
| ------------- | ---
| Branch?       | master
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | -
| License       | MIT
| Doc PR        | -

### CODE REVIEW

**Review by chalasr — APPROVED** [review]

File: src/Symfony/Component/Messenger/Middleware/ValidationMiddleware.php [hunk 1]
```diff
@@ -28,6 +29,9 @@ public function __construct(ValidatorInterface $validator)
         $this->validator = $validator;
     }
 
+    /**
+     * @param Envelope $envelope
+     */
```
Comment (chalasr) [inline src/Symfony/Component/Messenger/Middleware/ValidationMiddleware.php]: should be backported to 4.1

**Review by fabpot — CHANGES_REQUESTED** [review]

File: src/Symfony/Component/Messenger/Middleware/Enhancers/ActivationMiddlewareDecorator.php [hunk 2]
```diff
@@ -35,7 +35,7 @@ public function __construct(MiddlewareInterface $inner, $activated)
     }
 
     /**
-     * @param Envelope $message
+     * @param Envelope $envelope
```
[the remark this answers is not in the document]
Comment (sroze) [reply src/Symfony/Component/Messenger/Middleware/Enhancers/ActivationMiddlewareDecorator.php]: Yep, exactly, we can't typehint (as the interface does not have any). That's the subtlety of the `EnvelopeAwareInterface`. Having the PhpDoc "forces" your IDE to know about the right object it gets.

**Review by sroze — APPROVED** [review]

File: src/Symfony/Component/Messenger/Middleware/ValidationMiddleware.php (hunk 1 above)
Comment (fabpot) [reply src/Symfony/Component/Messenger/Middleware/ValidationMiddleware.php]: I don't think we need to backport this, not worth it.

**Comment by fabpot** [comment]
Thank you @pamil.

**Pull request merged**